### ISSUE TITLE
Unexpected raw output appended to rendered Web/API/URL_Pattern_API page. Hydration related? “Uncaught SyntaxError: JSON.parse: expected ',' or '}' after property value in … JSON data” logged to console

### ISSUE THREAD

**Issue by Jack-Zhang-1314** [mdn/yari#6780]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/URL_Pattern_API

### What specific section or headline is this issue about?

NO. At the bottom It has something that it should not have. 

### What information was incorrect, unhelpful, or incomplete?

<details>
<summary>screen capture</summary>

![Web capture_27-7-2022_9427_developer mozilla org](https://user-images.githubusercontent.com/82551626/181138696-56fc6aa4-851d-4b15-bdb9-b722b40cfa89.jpeg)
</details>

### What did you expect to see?

No need for the excess content at the bottom.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/url_pattern_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/URL_Pattern_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/url_pattern_api/index.md
* Last commit: https://github.com/mdn/content/commit/c984759744ba726d4f79b57f73db39566b711f84
* Document last modified: 2022-07-12T19:25:47.000Z

</details>

**Comment by sideshowbarker** [comment]
Confirmed that this is reproducible in all browsers in the production site — but not reproducible at all locally.

In the devtools console, among the error messages logged is this one:

>  Uncaught SyntaxError: JSON.parse: expected ',' or '}' after property value in object at line 1 column 15670 of the JSON data

…at index.tsx:22:9, which is the `JSON.parse()` call in this part:

```typescript
const hydrationElement = document.getElementById("hydration");
const appData = hydrationElement
  ? JSON.parse(hydrationElement.textContent!)
  : {};
```

**Comment by mirunacurtean** [comment]
Other examples:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
https://developer.mozilla.org/en-US/docs/Web/API/URL_Pattern_API#regex_matchers_limitations

**Comment by fiji-flo** [comment]
Will be fixed with #6782 (nasty one)

**Comment by fiji-flo** [comment]
Deployed and fixed! Thank you!
